Search intent opens onscreenkeyboard instead of using query.

### Describe the bug

I'm ever lasting integrating stuff to my home assistant. and now im connecting my android tv to my HA voice assist.
I want to make a jellyfin search but its not going as i hoped. 


`am start -a " android.search.action.GLOBAL_SEARCH" --es query "elfenlied" -n
    "org.jellyfin.androidtv/.ui.startup.StartupActivity"` starts jellyfin and opens the onscreen keyboard instead of applying the query

```service: androidtv.adb_command
data:
  command: >-
    am start -a " android.search.action.GLOBAL_SEARCH" --es query "elfenlied" -n
    "org.jellyfin.androidtv/.ui.startup.StartupActivity"
target:
  entity_id: media_player.android_tv_192_168_5_33```

### Logs

message from app
03-11 22:35:42.332 27236 27236 E cast_shell: [27236:27236:ERROR:legacy_app_id_mapper_impl.cc(58)] Unknown app_id AndroidNativeApp
03-11 22:35:42.336 27236 27236 E cast_shell: [27236:27236:ERROR:legacy_app_id_mapper_impl.cc(58)] Unknown app_id AndroidNativeApp
03-11 22:35:42.339 27236 27236 I cast_shell: [27236:27236:INFO:application_message_handler.cc(1716)] Application doesn't support playback device status
03-11 22:35:42.353 27236 27236 I cast_shell: [27236:27236:INFO:application_manager_impl.cc(2093)] Focusing foreground application: session=bba1d97e-94dd-48ca-914f-2c6c1496abe3
03-11 22:35:42.353 27236 27236 I cast_shell: [27236:27236:INFO:media_trait_handler_impl.cc(879)] Sent untracked GET_STATUS to app: session_id=bba1d97e-94dd-48ca-914f-2c6c1496abe3, request_id=1559774217
03-11 22:35:42.356 27236 27236 I cast_shell: [27236:27236:INFO:media_classifier_manager.cc(159)] app_id: AndroidNativeApp, is_spoken_word_app: 0
03-11 22:35:42.356 27236 27236 I cast_shell: [27236:27236:INFO:media_classifier_manager.cc(169)] app_id: AndroidNativeApp, is_movie_app: 0
03-11 22:35:42.357 27236 27236 W cast_shell: [27236:27236:WARNING:media_control_handler.cc(59)] Unknown player state: 
03-11 22:35:42.357 27236 27236 I cast_shell: [27236:27236:INFO:application_manager_impl.cc(1763)] App running: AndroidNativeApp (Jellyfin), session_id=bba1d97e-94dd-48ca-914f-2c6c1496abe3
03-11 22:35:42.358 27236 27236 E cast_shell: [27236:27236:ERROR:legacy_app_id_mapper_impl.cc(58)] Unknown app_id AndroidNativeApp
03-11 22:35:42.358 27236 27236 E cast_shell: [27236:27236:ERROR:legacy_app_id_mapper_impl.cc(58)] Unknown app_id AndroidNativeApp
03-11 22:35:42.363 27236 27429 I cast_shell: [27236:27429:INFO:metrics_recorder.cc(193)] Cast v2 action(socket_id=cc8d0851-11da-c234-6712-8de60b6cc94b.259): CastV2.Connection.Connect.In
03-11 22:35:42.366 27236 27429 I cast_shell: [27236:27429:INFO:metrics_recorder.cc(193)] Cast v2 action(socket_id=cc8d0851-11da-c234-6712-8de60b6cc94b.258): CastV2.Connection.Connect.In
03-11 22:35:42.368 27236 27429 I cast_shell: [27236:27429:INFO:metrics_recorder.cc(193)] Cast v2 action(socket_id=local-client.350): CastV2.Connection.Connect.In
03-11 22:35:42.369 27236 27429 I cast_shell: [27236:27429:INFO:metrics_recorder.cc(193)] Cast v2 action(socket_id=local-client.351): CastV2.Connection.Connect.In
03-11 22:35:42.371 27236 27429 I cast_shell: [27236:27429:INFO:metrics_recorder.cc(193)] Cast v2 action(socket_id=local-client.352): CastV2.Connection.Connect.In
03-11 22:35:42.371 27236 27236 E cast_shell: [27236:27236:ERROR:legacy_app_id_mapper_impl.cc(58)] Unknown app_id AndroidNativeApp
03-11 22:35:42.374 27236 27236 E cast_shell: [27236:27236:ERROR:legacy_app_id_mapper_impl.cc(58)] Unknown app_id AndroidNativeApp
03-11 22:35:42.374 27236 27429 I cast_shell: [27236:27429:INFO:metrics_recorder.cc(193)] Cast v2 action(socket_id=local-client.353): CastV2.Connection.Connect.In
03-11 22:35:42.375 27236 27236 I cast_shell: [27236:27236:INFO:metrics_recorder.cc(193)] cast event: CastV2.Ipc.Sender.Connected
03-11 22:35:42.375 27236 27236 I cast_shell: [27236:27236:INFO:application_message_handler.cc(545)] Strong connection connected: conn=bba1d97e-94dd-48ca-914f-2c6c1496abe3:cc8d0851-11da-c234-6712-8de60b6cc94b.259:sender-0, app=bba1d97e-94dd-48ca-914f-2c6c1496abe3
03-11 22:35:42.376 27236 27236 E cast_shell: [27236:27236:ERROR:legacy_app_id_mapper_impl.cc(58)] Unknown app_id AndroidNativeApp
03-11 22:35:42.376 27236 27429 I cast_shell: [27236:27429:INFO:metrics_recorder.cc(193)] Cast v2 action(socket_id=local-client.354): CastV2.Connection.Connect.In
03-11 22:35:42.377 27236 27429 I cast_shell: [27236:27429:INFO:v2_transport.cc(1205)] Heartbeat for V2TransportImpl started: max_inactivity=10000000, heartbeat_interval_=5000000
03-11 22:35:42.379 27236 27236 E cast_shell: [27236:27236:ERROR:legacy_app_id_mapper_impl.cc(58)] Unknown app_id AndroidNativeApp
03-11 22:35:42.379 27236 27236 I cast_shell: [27236:27236:INFO:metrics_recorder.cc(193)] cast event: CastV2.Ipc.Sender.Connected
03-11 22:35:42.379 27236 27236 I cast_shell: [27236:27236:INFO:application_message_handler.cc(545)] Strong connection connected: conn=bba1d97e-94dd-48ca-914f-2c6c1496abe3:cc8d0851-11da-c234-6712-8de60b6cc94b.258:sender-0, app=bba1d97e-94dd-48ca-914f-2c6c1496abe3
03-11 22:35:42.379 27236 27236 E cast_shell: [27236:27236:ERROR:legacy_app_id_mapper_impl.cc(58)] Unknown app_id AndroidNativeApp
03-11 22:35:42.380 27236 27236 E cast_shell: [27236:27236:ERROR:legacy_app_id_mapper_impl.cc(58)] Unknown app_id AndroidNativeApp
03-11 22:35:42.380 27236 27236 I cast_shell: [27236:27236:INFO:metrics_recorder.cc(193)] cast event: CastV2.Ipc.Sender.Connected
03-11 22:35:42.380 27236 27236 I cast_shell: [27236:27236:INFO:application_message_handler.cc(540)] Invisible connection connected: conn=bba1d97e-94dd-48ca-914f-2c6c1496abe3:local-client.350:mediacontrol-0, app=bba1d97e-94dd-48ca-914f-2c6c1496abe3
03-11 22:35:42.380 27236 27236 E cast_shell: [27236:27236:ERROR:legacy_app_id_mapper_impl.cc(58)] Unknown app_id AndroidNativeApp
03-11 22:35:42.380 27236 27236 I cast_shell: [27236:27236:INFO:metrics_recorder.cc(193)] cast event: CastV2.Ipc.Sender.Connected
03-11 22:35:42.381 27236 27236 I cast_shell: [27236:27236:INFO:application_message_handler.cc(540)] Invisible connection connected: conn=bba1d97e-94dd-48ca-914f-2c6c1496abe3:local-client.351:mediacontrol-0, app=bba1d97e-94dd-48ca-914f-2c6c1496abe3
03-11 22:35:42.381 27236 27236 E cast_shell: [27236:27236:ERROR:legacy_app_id_mapper_impl.cc(58)] Unknown app_id AndroidNativeApp
03-11 22:35:42.381 27236 27236 I cast_shell: [27236:27236:INFO:metrics_recorder.cc(193)] cast event: CastV2.Ipc.Sender.Connected
03-11 22:35:42.381 27236 27236 I cast_shell: [27236:27236:INFO:application_message_handler.cc(540)] Invisible connection connected: conn=bba1d97e-94dd-48ca-914f-2c6c1496abe3:local-client.352:mediacontrol-0, app=bba1d97e-94dd-48ca-914f-2c6c1496abe3
03-11 22:35:42.382 27236 27236 E cast_shell: [27236:27236:ERROR:legacy_app_id_mapper_impl.cc(58)] Unknown app_id AndroidNativeApp
03-11 22:35:42.383 27236 27236 I cast_shell: [27236:27236:INFO:metrics_recorder.cc(193)] cast event: CastV2.Ipc.Sender.Connected
03-11 22:35:42.383 27236 27236 I cast_shell: [27236:27236:INFO:application_message_handler.cc(540)] Invisible connection connected: conn=bba1d97e-94dd-48ca-914f-2c6c1496abe3:local-client.353:mediacontrol-0, app=bba1d97e-94dd-48ca-914f-2c6c1496abe3
03-11 22:35:42.384 27236 27236 E cast_shell: [27236:27236:ERROR:legacy_app_id_mapper_impl.cc(58)] Unknown app_id AndroidNativeApp
03-11 22:35:42.384 27236 27236 I cast_shell: [27236:27236:INFO:metrics_recorder.cc(193)] cast event: CastV2.Ipc.Sender.Connected
03-11 22:35:42.384 27236 27236 I cast_shell: [27236:27236:INFO:application_message_handler.cc(540)] Invisible connection connected: conn=bba1d97e-94dd-48ca-914f-2c6c1496abe3:local-client.354:mediacontrol-0, app=bba1d97e-94dd-48ca-914f-2c6c1496abe3
03-11 22:35:42.388 27236 27236 W WargReceiverApp: Received a message of unsupported namespace: urn:x-cast:com.google.cast.system
03-11 22:35:42.394 27236 27236 W WargReceiverApp: Received a message of unsupported namespace: urn:x-cast:com.google.cast.system
03-11 22:35:42.399 27236 27236 W BasicMediaStatusFactory: org.jellyfin.androidtv's media session has no metadata
03-11 22:35:42.403 27236 27236 D BasicMediaStatusFactory: Supported media session actions: 7095247
03-11 22:35:42.403 27236 27236 W WargReceiverApp: Received a message of unsupported namespace: urn:x-cast:com.google.cast.system
03-11 22:35:42.403 27236 27236 W WargReceiverApp: Received a message of unsupported namespace: urn:x-cast:com.google.cast.system
03-11 22:35:42.404 27236 27236 W BasicMediaStatusFactory: org.jellyfin.androidtv's media session has no metadata
03-11 22:35:42.405 27236 27236 D BasicMediaStatusFactory: Supported media session actions: 7095247
03-11 22:35:42.406 27236 27236 W BasicMediaStatusFactory: org.jellyfin.androidtv's media session has no metadata
03-11 22:35:42.407 27236 27236 D BasicMediaStatusFactory: Supported media session actions: 7095247
03-11 22:35:42.410 27236 27236 W cast_shell: [27236:27236:WARNING:media_trait_handler_impl.cc(678)] No media status
03-11 22:35:42.414 27236 27236 W cast_shell: [27236:27236:WARNING:media_trait_handler_impl.cc(678)] No media status
03-11 22:35:42.452 24694 24705 I chatty  : uid=10076(org.jellyfin.androidtv) HeapTaskDaemon expire 1 line
03-11 22:35:42.540 24694 24863 I chatty  : uid=10076(org.jellyfin.androidtv) DefaultDispatch expire 6 lines
03-11 22:35:42.646  3606  3783 D DataMonitor: onChange
03-11 22:35:42.646  3606  3783 D ActionDebounce: debounce Action skip last
03-11 22:35:42.683  2567  2619 W         : buffer descriptor with invalid usage bits 0x400
03-11 22:35:42.701  2567  2619 I chatty  : uid=1000(system) Binder:2567_2 identical 2 lines
03-11 22:35:42.716  2567  2619 W         : buffer descriptor with invalid usage bits 0x400
03-11 22:35:42.737  3606  3783 D DataMonitor: onChange
03-11 22:35:42.737  3606  3783 D ActionDebounce: debounce Action skip last
03-11 22:35:42.769  3606  3783 D DataMonitor: onChange
03-11 22:35:42.769  3606  3783 D ActionDebounce: debounce Action skip last
03-11 22:35:42.814  3606  3783 D DataMonitor: onChange
03-11 22:35:42.814  3606  3783 D ActionDebounce: debounce Action skip last
03-11 22:35:42.822  3606  3783 D DataMonitor: onChange
03-11 22:35:42.822  3606  3783 D ActionDebounce: debounce Action skip last
03-11 22:35:42.860 24694 24694 I chatty  : uid=10076(org.jellyfin.androidtv) expire 22 lines
03-11 22:35:42.878  3606  3783 D DataMonitor: onChange
03-11 22:35:42.878  3606  3783 D ActionDebounce: debounce Action skip last
03-11 22:35:42.893  3606  3783 D DataMonitor: onChange
03-11 22:35:42.893  3606  3783 D ActionDebounce: debounce Action skip last
03-11 22:35:42.914  3606  3783 D DataMonitor: onChange
03-11 22:35:42.914  3606  3783 D ActionDebounce: debounce Action skip last
03-11 22:35:42.956  3606  3783 D DataMonitor: onChange
03-11 22:35:42.956  3606  3783 D ActionDebounce: debounce Action skip last
03-11 22:35:43.007  3606  3783 D DataMonitor: onChange
03-11 22:35:43.007  3606  3783 D ActionDebounce: debounce Action skip last
03-11 22:35:43.131 24694 24705 I chatty  : uid=10076(org.jellyfin.androidtv) HeapTaskDaemon expire 3 lines
03-11 22:35:43.208 24694 24792 I chatty  : uid=10076(org.jellyfin.androidtv) RenderThread expire 3 lines
03-11 22:35:43.261 20960 20960 I GoogleInputMethodService: GoogleInputMethodService.onFinishInput():3264 
03-11 22:35:43.277 20960 20960 I GoogleInputMethodService: GoogleInputMethodService.updateDeviceLockedStatus():2111 repeatCheckTimes = 0, locked = false
03-11 22:35:43.277 20960 20960 I GoogleInputMethodService: GoogleInputMethodService.onStartInput():1899 onStartInput(EditorInfo{inputType=0x0(NULL) imeOptions=0x0 privateImeOptions=null actionName=UNSPECIFIED actionLabel=null actionId=0 initialSelStart=-1 initialSelEnd=-1 initialCapsMode=0x0 hintText=null label=null packageName=org.jellyfin.androidtv fieldId=2131427951 fieldName=null extras=null hintLocales=[]}, false)
03-11 22:35:43.277 20960 20960 I GoogleInputMethodService: GoogleInputMethodService.shouldHideHeaderOnInitialState():4091 ShouldHideHeaderOnInitialState = false
03-11 22:35:43.279 20960 20960 I GoogleInputMethodService: GoogleInputMethodService.updateDeviceLockedStatus():2111 repeatCheckTimes = 1, locked = false
03-11 22:35:43.338 19309 27176 D WireGuard/GoBackend/SurfsharkWireguardTunnel: peer(qFuw…TxUs) - Sending keepalive packet
03-11 22:35:43.428  2725 27235 D DrmHal  : found drm@1.0 IDrmFactory clearkey
03-11 22:35:43.439  2725 27235 D DrmHal  : found drm@1.0 IDrmFactory default
03-11 22:35:43.444  2725 27235 D DrmHal  : found drm@1.0 IDrmFactory widevine
03-11 22:35:43.457  2725 27235 D DrmHal  : found drm@1.1 IDrmFactory clearkey
03-11 22:35:43.460  2725 27235 D DrmHal  : found drm@1.1 IDrmFactory widevine
03-11 22:35:43.477  2543  2543 V PlayReadyDrmFactory: isCryptoSchemeSupported >>
03-11 22:35:43.477  2543  2543 I PlayReadyDrmFactory: PlayReady Plugin version = PRAN-3X.00.20190528
03-11 22:35:43.479  2543  2543 D NetflixDrmPlugin: uuid:0xedef8ba979d64acea3c827dcd51d21ed
03-11 22:35:43.479  2543  2543 D NetflixDrmPlugin: [isCryptoSchemeSupported:65] return false
03-11 22:35:43.505  2543  2543 V PlayReadyDrmFactory: isCryptoSchemeSupported >>
03-11 22:35:43.505  2543  2543 I PlayReadyDrmFactory: PlayReady Plugin version = PRAN-3X.00.20190528
03-11 22:35:43.505  2543  2543 D NetflixDrmPlugin: uuid:0xedef8ba979d64acea3c827dcd51d21ed
03-11 22:35:43.505  2543  2543 D NetflixDrmPlugin: [isCryptoSchemeSupported:65] return false
03-11 22:35:43.507  2543  2543 D WVCdm   : Instantiating CDM.
03-11 22:35:43.534  2545  2545 D WVCdm   : Instantiating CDM.
03-11 22:35:43.546  2545  5120 I WVCdm   : [cdm_engine.cpp(484):QueryStatus] CdmEngine::QueryStatus
03-11 22:35:43.560  2545 24876 I WVCdm   : [(0):] Level3 Library 4445 Oct  3 2018 13:10:36
03-11 22:35:43.631  2545  5120 I WVCdm   : [oemcrypto_adapter_dynamic.cpp(573):Initialize] L3 Initialized. Trying L1.
03-11 22:35:43.701  2545  5120 I WV_CA_OEM: [Info] Widevine V14 CA, Version: "WV-14.10.eb5a0fd.2019040222", Mstar commit:"eb5a0fdd14f4da35f5c12857edff2aced11f2fa8" Build Time: 22:20:07 Apr  2 2019
03-11 22:35:43.711  2545  5120 I WV_CA_OEM: [TeeProxy::TeeProxy()] Touch a file at /Customer/OPTEE_UUID for recording UUID, but fopen FAILED 
03-11 22:35:43.712  2545  5120 I WV_CA_OEM: Widevine CA TA version check CA: Oem v14.1 TA: Oem v14.1 
03-11 22:35:43.713  2545  5120 D WVCdm   : [oemcrypto_adapter_dynamic.cpp(590):Initialize] OEMCrypto_Initialize Level 1 success. I will use level 1.
03-11 22:35:43.723  2545  5120 D WVCdm   : [crypto_session.cpp(668):Open] CryptoSession::Open: Lock: requested_security_level: Default
03-11 22:35:43.747  2545  5120 I WVCdm   : [(0):] L3 Terminate.
03-11 22:35:43.769  2545  5120 I WVCdm   : [cdm_engine.cpp(112):OpenSession] CdmEngine::OpenSession
03-11 22:35:43.770  2545 24877 I WVCdm   : [(0):] Level3 Library 4445 Oct  3 2018 13:10:36
03-11 22:35:43.792  2545  5120 I WVCdm   : [oemcrypto_adapter_dynamic.cpp(573):Initialize] L3 Initialized. Trying L1.
03-11 22:35:43.796  2545  5120 I WV_CA_OEM: [Info] Widevine V14 CA, Version: "WV-14.10.eb5a0fd.2019040222", Mstar commit:"eb5a0fdd14f4da35f5c12857edff2aced11f2fa8" Build Time: 22:20:07 Apr  2 2019
03-11 22:35:43.797  2545  5120 I WV_CA_OEM: [TeeProxy::TeeProxy()] Touch a file at /Customer/OPTEE_UUID for recording UUID, but fopen FAILED 
03-11 22:35:43.797  2545  5120 I WV_CA_OEM: Widevine CA TA version check CA: Oem v14.1 TA: Oem v14.1 
03-11 22:35:43.799  2545  5120 D WVCdm   : [oemcrypto_adapter_dynamic.cpp(590):Initialize] OEMCrypto_Initialize Level 1 success. I will use level 1.
03-11 22:35:43.799  2545  5120 D WVCdm   : [crypto_session.cpp(668):Open] CryptoSession::Open: Lock: requested_security_level: Default
03-11 22:35:43.802  2545  5120 I WVCdm   : [device_files.cpp(166):ExtractDeviceInfo] ExtractDeviceInfo Entry
03-11 22:35:43.806  2545  5120 I WVCdm   : [cdm_engine.cpp(149):OpenSession] CdmEngine::OpenSession: sid329
03-11 22:35:43.806  2545  5120 I WVCdm   : [cdm_engine.cpp(754):QueryOemCryptoSessionId] CdmEngine::QueryOemCryptoSessionId
03-11 22:35:43.807 27236 27360 I cast_shell: [27236:27360:INFO:jni_client_auth_creds_mediadrm.cc(256)] Cast: signing message with length 51
03-11 22:35:43.864  2545  2545 I WVCdm   : [cdm_engine.cpp(192):CloseSession] CdmEngine::CloseSession: sid329
03-11 22:35:43.876  2545  2545 I WVCdm   : [(0):] L3 Terminate.
03-11 22:35:43.892 27236 27236 I cast_shell: [27236:27236:INFO:application_identification_settings.cc(1079)] CreateJwt completed; running all done signing callbacks.
03-11 22:35:43.893 27236 27236 I cast_shell: [27236:27236:INFO:metrics_recorder.cc(193)] cast event: Cloud.Transport.Https.StatusChange.Out
03-11 22:35:43.944 24694 24849 I chatty  : uid=10076(org.jellyfin.androidtv) WM.task-3 expire 1 line
03-11 22:35:44.008  3606  3783 D DataAccessManager: syncNids
03-11 22:35:44.010  3606  3783 V DataAccessManager: isFvpAuthConfigEnabled status=2
03-11 22:35:44.160 27236 27245 W System  : A resource failed to call release. 
03-11 22:35:44.267 27236 27428 I cast_shell: [27236:27428:INFO:mdns_app_filter.cc(1860)] MdnsAppFilter: Done announcing.
03-11 22:35:44.492 27236 27236 I cast_shell: [27236:27236:INFO:metrics_recorder.cc(193)] cast event: Cloud.Transport.Https.StatusChange.Out.Success
03-11 22:35:44.536 20960 20960 I GoogleInputMethodService: GoogleInputMethodService.onFinishInput():3264 
03-11 22:35:44.538 20960 20960 I GoogleInputMethodService: GoogleInputMethodService.updateDeviceLockedStatus():2111 repeatCheckTimes = 0, locked = false
03-11 22:35:44.538 20960 20960 I GoogleInputMethodService: GoogleInputMethodService.onStartInput():1899 onStartInput(EditorInfo{inputType=0x0(NULL) imeOptions=0x0 privateImeOptions=null actionName=UNSPECIFIED actionLabel=null actionId=0 initialSelStart=-1 initialSelEnd=-1 initialCapsMode=0x0 hintText=null label=null packageName=org.jellyfin.androidtv fieldId=-1 fieldName=null extras=null hintLocales=[]}, false)
03-11 22:35:44.538 20960 20960 I GoogleInputMethodService: GoogleInputMethodService.shouldHideHeaderOnInitialState():4091 ShouldHideHeaderOnInitialState = false
03-11 22:35:44.539 20960 20960 I GoogleInputMethodService: GoogleInputMethodService.updateDeviceLockedStatus():2111 repeatCheckTimes = 1, locked = false
03-11 22:35:44.623 24694 24879 I chatty  : uid=10076(org.jellyfin.androidtv) expire 1 line
03-11 22:35:44.748  2728 13754 I AlarmManager: Setting Wakeup Alarm type[2] sPackageName is null @1936985056 and next wakeup alarm is @Tue Mar 12 01:33:38 GMT+01:00 2024
03-11 22:35:45.050 24694 24845 I chatty  : uid=10076(org.jellyfin.androidtv) DefaultDispatch expire 111 lines
03-11 22:35:45.056 24694 24845 I lyfin.androidt:   at void coil.disk.DiskLruCache.initialize() (DiskLruCache.kt:179)
03-11 22:35:45.056 24694 24845 I lyfin.androidt:   at coil.disk.DiskLruCache$Snapshot coil.disk.DiskLruCache.get(java.lang.String) (DiskLruCache.kt:365)
03-11 22:35:45.057 24694 24845 I lyfin.androidt:   at coil.disk.DiskCache$Snapshot coil.disk.RealDiskCache.openSnapshot(java.lang.String) (RealDiskCache.kt:29)
03-11 22:35:45.057 24694 24845 I lyfin.androidt:   at coil.disk.DiskCache$Snapshot coil.fetch.HttpUriFetcher.readFromDiskCache() (HttpUriFetcher.kt:127)
03-11 22:35:45.057 24694 24845 I lyfin.androidt:   at java.lang.Object coil.fetch.HttpUriFetcher.fetch(kotlin.coroutines.Continuation) (HttpUriFetcher.kt:40)
03-11 22:35:45.057 24694 24845 I lyfin.androidt:   at java.lang.Object coil.intercept.EngineInterceptor.fetch(coil.ComponentRegistry, coil.request.ImageRequest, java.lang.Object, coil.request.Options, coil.EventListener, kotlin.coroutines.Continuation) (EngineInterceptor.kt:165)
03-11 22:35:45.057 24694 24845 I lyfin.androidt:   at java.lang.Object coil.intercept.EngineInterceptor.execute(coil.request.ImageRequest, java.lang.Object, coil.request.Options, coil.EventListener, kotlin.coroutines.Continuation) (EngineInterceptor.kt:122)
03-11 22:35:45.057 24694 24845 I lyfin.androidt:   at java.lang.Object coil.intercept.EngineInterceptor.access$execute(coil.intercept.EngineInterceptor, coil.request.ImageRequest, java.lang.Object, coil.request.Options, coil.EventListener, kotlin.coroutines.Continuation) (EngineInterceptor.kt:41)
03-11 22:35:45.057 24694 24845 I lyfin.androidt:   at java.lang.Object coil.intercept.EngineInterceptor$intercept$2.invokeSuspend(java.lang.Object) (EngineInterceptor.kt:75)
03-11 22:35:45.057 24694 24845 I lyfin.androidt:   at void kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(java.lang.Object) (ContinuationImpl.kt:33)
03-11 22:35:45.057 24694 24845 I lyfin.androidt:   at void kotlinx.coroutines.DispatchedTask.run() (DispatchedTask.kt:108)
03-11 22:35:45.057 24694 24845 I lyfin.androidt:   at void kotlinx.coroutines.internal.LimitedDispatcher$Worker.run() (LimitedDispatcher.kt:115)
03-11 22:35:45.057 24694 24845 I lyfin.androidt:   at void kotlinx.coroutines.scheduling.TaskImpl.run() (Tasks.kt:103)
03-11 22:35:45.057 24694 24845 I lyfin.androidt:   at void kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(kotlinx.coroutines.scheduling.Task) (CoroutineScheduler.kt:584)
03-11 22:35:45.057 24694 24845 I lyfin.androidt:   at void kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(kotlinx.coroutines.scheduling.Task) (CoroutineScheduler.kt:793)
03-11 22:35:45.057 24694 24845 I lyfin.androidt:   at void kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker() (CoroutineScheduler.kt:697)
03-11 22:35:45.057 24694 24845 I lyfin.androidt:   at void kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run() (CoroutineScheduler.kt:684)
03-11 22:35:45.057 24694 24845 I lyfin.androidt: Caused by: java.lang.ClassNotFoundException: Didn't find class "java.lang.ClassValue" on path: DexPathList[[zip file "/data/app/org.jellyfin.androidtv-_WU90Nwqdp2BbyS1cTqa3g==/base.apk", zip file "/data/app/org.jellyfin.androidtv-_WU90Nwqdp2BbyS1cTqa3g==/split_config.armeabi_v7a.apk", zip file "/data/app/org.jellyfin.androidtv-_WU90Nwqdp2BbyS1cTqa3g==/split_config.nl.apk", zip file "/data/app/org.jellyfin.androidtv-_WU90Nwqdp2BbyS1cTqa3g==/split_config.tvdpi.apk"],nativeLibraryDirectories=[/data/app/org.jellyfin.androidtv-_WU90Nwqdp2BbyS1cTqa3g==/lib/arm, /data/app/org.jellyfin.androidtv-_WU90Nwqdp2BbyS1cTqa3g==/base.apk!/lib/armeabi-v7a, /data/app/org.jellyfin.androidtv-_WU90Nwqdp2BbyS1cTqa3g==/split_config.armeabi_v7a.apk!/lib/armeabi-v7a, /data/app/org.jellyfin.androidtv-_WU90Nwqdp2BbyS1cTqa3g==/split_config.nl.apk!/lib/armeabi-v7a, /data/app/org.jellyfin.androidtv-_WU90Nwqdp2BbyS1cTqa3g==/split_config.tvdpi.apk!/lib/armeabi-v7a, /system/lib, /system/product/lib]]
03-11 22:35:45.057 24694 24845 I lyfin.androidt:   at java.lang.Class dalvik.system.BaseDexClassLoader.findClass(java.lang.String) (BaseDexClassLoader.java:134)
03-11 22:35:45.057 24694 24845 I lyfin.androidt:   at java.lang.Class java.lang.ClassLoader.loadClass(java.lang.String, boolean) (ClassLoader.java:379)
03-11 22:35:45.057 24694 24845 I lyfin.androidt:   at java.lang.Class java.lang.ClassLoader.loadClass(java.lang.String) (ClassLoader.java:312)
03-11 22:35:45.057 24694 24845 I lyfin.androidt:   at boolean org.acra.util.BundleKt$$ExternalSyntheticApiModelOutline0.m(java.nio.file.attribute.BasicFileAttributes) (D8$$SyntheticClass:-1)
03-11 22:35:45.057 24694 24845 I lyfin.androidt:   at okio.FileMetadata okio.NioSystemFileSystem.metadataOrNull(java.nio.file.Path) (NioSystemFileSystem.kt:58)
03-11 22:35:45.057 24694 24845 I lyfin.androidt:   at okio.FileMetadata okio.NioSystemFileSystem.metadataOrNull(okio.Path) (NioSystemFileSystem.kt:35)
03-11 22:35:45.058 24694 24845 I lyfin.androidt:   at okio.FileMetadata okio.ForwardingFileSystem.metadataOrNull(okio.Path) (ForwardingFileSystem.kt:155)
03-11 22:35:45.058 24694 24845 I lyfin.androidt:   at boolean okio.internal.-FileSystem.commonExists(okio.FileSystem, okio.Path) (FileSystem.kt:41)
03-11 22:35:45.058 24694 24845 I lyfin.androidt:   at boolean okio.FileSystem.exists(okio.Path) (FileSystem.kt:39)
03-11 22:35:45.058 24694 24845 I lyfin.androidt:   at void coil.disk.DiskLruCache.initialize() (DiskLruCache.kt:179)
03-11 22:35:45.058 24694 24845 I lyfin.androidt:   at coil.disk.DiskLruCache$Snapshot coil.disk.DiskLruCache.get(java.lang.String) (DiskLruCache.kt:365)
03-11 22:35:45.058 24694 24845 I lyfin.androidt:   at coil.disk.DiskCache$Snapshot coil.disk.RealDiskCache.openSnapshot(java.lang.String) (RealDiskCache.kt:29)
03-11 22:35:45.058 24694 24845 I lyfin.androidt:   at coil.disk.DiskCache$Snapshot coil.fetch.HttpUriFetcher.readFromDiskCache() (HttpUriFetcher.kt:127)
03-11 22:35:45.058 24694 24845 I lyfin.androidt:   at java.lang.Object coil.fetch.HttpUriFetcher.fetch(kotlin.coroutines.Continuation) (HttpUriFetcher.kt:40)
03-11 22:35:45.058 24694 24845 I lyfin.androidt:   at java.lang.Object coil.intercept.EngineInterceptor.fetch(coil.ComponentRegistry, coil.request.ImageRequest, java.lang.Object, coil.request.Options, coil.EventListener, kotlin.coroutines.Continuation) (EngineInterceptor.kt:165)
03-11 22:35:45.058 24694 24845 I lyfin.androidt:   at java.lang.Object coil.intercept.EngineInterceptor.execute(coil.request.ImageRequest, java.lang.Object, coil.request.Options, coil.EventListener, kotlin.coroutines.Continuation) (EngineInterceptor.kt:122)
03-11 22:35:45.058 24694 24845 I lyfin.androidt:   at java.lang.Object coil.intercept.EngineInterceptor.access$execute(coil.intercept.EngineInterceptor, coil.request.ImageRequest, java.lang.Object, coil.request.Options, coil.EventListener, kotlin.coroutines.Continuation) (EngineInterceptor.kt:41)
03-11 22:35:45.058 24694 24845 I lyfin.androidt:   at java.lang.Object coil.intercept.EngineInterceptor$intercept$2.invokeSuspend(java.lang.Object) (EngineInterceptor.kt:75)
03-11 22:35:45.058 24694 24845 I lyfin.androidt:   at void kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(java.lang.Object) (ContinuationImpl.kt:33)
03-11 22:35:45.058 24694 24845 I lyfin.androidt:   at void kotlinx.coroutines.DispatchedTask.run() (DispatchedTask.kt:108)
03-11 22:35:45.058 24694 24845 I lyfin.androidt:   at void kotlinx.coroutines.internal.LimitedDispatcher$Worker.run() (LimitedDispatcher.kt:115)
03-11 22:35:45.058 24694 24845 I lyfin.androidt:   at void kotlinx.coroutines.scheduling.TaskImpl.run() (Tasks.kt:103)
03-11 22:35:45.058 24694 24845 I lyfin.androidt:   at void kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(kotlinx.coroutines.scheduling.Task) (CoroutineScheduler.kt:584)
03-11 22:35:45.058 24694 24845 I lyfin.androidt:   at void kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(kotlinx.coroutines.scheduling.Task) (CoroutineScheduler.kt:793)
03-11 22:35:45.058 24694 24845 I lyfin.androidt:   at void kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker() (CoroutineScheduler.kt:697)
03-11 22:35:45.058 24694 24845 I lyfin.androidt:   at void kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run() (CoroutineScheduler.kt:684)
03-11 22:35:45.058 24694 24845 I lyfin.androidt: 
03-11 22:35:45.237  2728  3418 E TaskPersister: File error accessing recents directory (directory doesn't exist?).
03-11 22:35:45.277 24694 24705 I lyfin.androidt: Background concurrent copying GC freed 49440(2MB) AllocSpace objects, 5(112KB) LOS objects, 25% free, 6MB/8MB, paused 199us total 332.221ms
03-11 22:35:45.497  2728  2752 I system_server: Background concurrent copying GC freed 60940(3MB) AllocSpace objects, 20(720KB) LOS objects, 19% free, 11MB/14MB, paused 535us total 342.645ms
03-11 22:35:45.667 21181 21181 D BoundBrokerSvc: onUnbind: Intent { act=com.google.android.mobstore.service.START pkg=com.google.android.gms }
03-11 22:35:45.958 24694 24705 I lyfin.androidt: Background concurrent copying GC freed 13900(787KB) AllocSpace objects, 0(0B) LOS objects, 25% free, 9MB/12MB, paused 507us total 341.737ms
03-11 22:35:46.269 24694 24694 I AutoBitrate: Auto bitrate set to: 5752291
03-11 22:35:46.368 24694 24705 I lyfin.androidt: Background concurrent copying GC freed 13696(831KB) AllocSpace objects, 14(280KB) LOS objects, 25% free, 13MB/18MB, paused 256us total 169.919ms
03-11 22:35:49.773 21181 21181 D BoundBrokerSvc: onUnbind: Intent { act=com.google.android.gms.measurement.START pkg=com.google.android.gms }
03-11 22:35:50.234 27236 27236 I cast_shell: [27236:27236:INFO:service_base.h(78)] Size of the map for type AuthTokenResponse is 0
03-11 22:35:50.236 27236 27236 I cast_shell: [27236:27236:INFO:service_base.h(246)] Container sizes for type CloudSetting are (session_info_map: 0), (topic_info_map: 21), (pending_events: 0).
03-11 22:35:50.238 27236 27236 I cast_shell: [27236:27236:INFO:service_base.h(78)] Size of the map for type LocalDeviceInfo is 0
03-11 22:35:50.238 27236 27236 I cast_shell: [27236:27236:INFO:service_base.h(78)] Size of the map for type NetworkInfo is 0
03-11 22:35:50.238 27236 27236 I cast_shell: [27236:27236:INFO:metrics_stat_logger.cc(54)] Metrics stat for the past 60 secs : total=18
03-11 22:35:50.238 27236 27236 I cast_shell: [metrics_stat_logger] AppReady=1
03-11 22:35:50.238 27236 27236 I cast_shell: [metrics_stat_logger] AppStarted=1
03-11 22:35:50.238 27236 27236 I cast_shell: [metrics_stat_logger] Cast.AppLaunchAuthorizer.AutoAuthorize=1
03-11 22:35:50.238 27236 27236 I cast_shell: [metrics_stat_logger] Cast.AppLaunchAuthorizer.TryAuthorizeLaunch=1
03-11 22:35:50.238 27236 27236 I cast_shell: [metrics_stat_logger] Cast.Discovery.Mdns.Query.Cast.Count=1
03-11 22:35:50.238 27236 27236 I cast_shell: [metrics_stat_logger] Cast.Discovery.Mdns.Request.In=2
03-11 22:35:50.238 27236 27236 I cast_shell: [metrics_stat_logger] Cast.Discovery.Mdns.Response.Out=2
03-11 22:35:50.238 27236 27236 I cast_shell: [metrics_stat_logger] Cast.Platform.TrimMemory=2
03-11 22:35:50.238 27236 27236 I cast_shell: [metrics_stat_logger] CastV2.Connection.Connect.In=7
03-11 22:35:50.238 27236 27236 I cast_shell: [metrics_stat_logger] CastV2.Ipc.Sender.Connected=7
03-11 22:35:50.238 27236 27236 I cast_shell: [metrics_stat_logger] CastV2.Ipc.System.In.Internal=8
03-11 22:35:50.238 27236 27236 I cast_shell: [metrics_stat_logger] CastV2.Ipc.System.Out.Internal=2
03-11 22:35:50.238 27236 27236 I cast_shell: [metrics_stat_logger] CastV2.Receiver.DeviceStatus.Out=8
03-11 22:35:50.238 27236 27236 I cast_shell: [metrics_stat_logger] CastV2.Receiver.DeviceStatus.Out.Broadcast=4
03-11 22:35:50.238 27236 27236 I cast_shell: [metrics_stat_logger] CastV2.Transport.Accept=5
03-11 22:35:50.238 27236 27236 I cast_shell: [metrics_stat_logger] CastV2.Transport.Close=3
03-11 22:35:50.238 27236 27236 I cast_shell: [metrics_stat_logger] Cloud.Transport.Https.StatusChange.Out=1
03-11 22:35:50.238 27236 27236 I cast_shell: [metrics_stat_logger] Cloud.Transport.Https.StatusChange.Out.Success=1
03-11 22:35:55.443  2739  2739 D MTK_KL  : <31>[44398.904814] logd: logdr: UID=2000 GID=2000 PID=24946 n tail=0 logMask=19 pid=0 start=0ns timeout=0ns


### Application version

0.16.6

### Where did you install the app from?

None

### Device information

cheap medion tv. MD30100

### Android version

9

### Jellyfin server version

10.8.13-1

--- a/app/src/main/java/org/jellyfin/androidtv/ui/home/HomeFragment.kt
+++ b/app/src/main/java/org/jellyfin/androidtv/ui/home/HomeFragment.kt
@@ -44,7 +44,7 @@ class HomeFragment : Fragment() {
 		}
 
 		binding.search.setOnClickListener {
-			navigationRepository.navigate(Destinations.search)
+			navigationRepository.navigate(Destinations.search())
 		}
 
 		return binding.root

--- a/app/src/main/java/org/jellyfin/androidtv/ui/navigation/Destinations.kt
+++ b/app/src/main/java/org/jellyfin/androidtv/ui/navigation/Destinations.kt
@@ -46,7 +46,9 @@ object Destinations {
 
 	// General
 	val home = fragmentDestination<HomeFragment>()
-	val search = fragmentDestination<SearchFragment>()
+	fun search(query: String? = null) = fragmentDestination<SearchFragment>(
+		SearchFragment.EXTRA_QUERY to query,
+	)
 	val userPreferences = preferenceDestination<UserPreferencesScreen>()
 
 	// Browsing

--- a/app/src/main/java/org/jellyfin/androidtv/ui/search/LeanbackSearchFragment.kt
+++ b/app/src/main/java/org/jellyfin/androidtv/ui/search/LeanbackSearchFragment.kt
@@ -31,6 +31,9 @@ class LeanbackSearchFragment : SearchSupportFragment(), SearchSupportFragment.Se
 		viewModel.searchResultsFlow
 			.onEach { searchFragmentDelegate.showResults(it) }
 			.launchIn(lifecycleScope)
+
+		val query = arguments?.getString(SearchFragment.EXTRA_QUERY)
+		if (!query.isNullOrBlank()) setSearchQuery(query, true)
 	}
 
 	override fun getResultsAdapter() = searchFragmentDelegate.rowsAdapter

--- a/app/src/main/java/org/jellyfin/androidtv/ui/search/SearchFragment.kt
+++ b/app/src/main/java/org/jellyfin/androidtv/ui/search/SearchFragment.kt
@@ -6,9 +6,14 @@ import android.os.Bundle
 import android.speech.SpeechRecognizer
 import androidx.core.content.ContextCompat
 import androidx.fragment.app.Fragment
+import androidx.fragment.app.commit
 import org.jellyfin.androidtv.R
 
 class SearchFragment : Fragment(R.layout.fragment_content_view) {
+	companion object {
+		const val EXTRA_QUERY = "query"
+	}
+
 	private val isSpeechEnabled by lazy {
 		SpeechRecognizer.isRecognitionAvailable(requireContext())
 			&& ContextCompat.checkSelfPermission(
@@ -22,14 +27,13 @@ class SearchFragment : Fragment(R.layout.fragment_content_view) {
 
 		// Determine fragment to use
 		val searchFragment = when {
-			isSpeechEnabled -> LeanbackSearchFragment()
-			else -> TextSearchFragment()
+			isSpeechEnabled -> LeanbackSearchFragment::class.java
+			else -> TextSearchFragment::class.java
 		}
 
 		// Add fragment
-		childFragmentManager
-			.beginTransaction()
-			.replace(R.id.content_view, searchFragment)
-			.commit()
+		childFragmentManager.commit {
+			replace(R.id.content_view, searchFragment, arguments)
+		}
 	}
 }

--- a/app/src/main/java/org/jellyfin/androidtv/ui/search/TextSearchFragment.kt
+++ b/app/src/main/java/org/jellyfin/androidtv/ui/search/TextSearchFragment.kt
@@ -11,6 +11,7 @@ import android.view.inputmethod.InputMethodManager
 import android.widget.EditText
 import androidx.core.content.getSystemService
 import androidx.fragment.app.Fragment
+import androidx.fragment.app.commit
 import androidx.leanback.app.RowsSupportFragment
 import androidx.lifecycle.lifecycleScope
 import kotlinx.coroutines.flow.launchIn
@@ -36,26 +37,40 @@ class TextSearchFragment : Fragment() {
 		savedInstanceState: Bundle?
 	): View {
 		_binding = FragmentSearchTextBinding.inflate(inflater, container, false)
-		return binding.root
-	}
-
-	override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
-		super.onViewCreated(view, savedInstanceState)
 
 		binding.searchBar.apply {
 			onTextChanged { viewModel.searchDebounced(it) }
 			onSubmit { viewModel.searchImmediately(it) }
 		}
 
-		binding.resultsFrame.getFragment<RowsSupportFragment?>()?.let {
-			it.adapter = searchFragmentDelegate.rowsAdapter
-			it.onItemViewClickedListener = searchFragmentDelegate.onItemViewClickedListener
-			it.onItemViewSelectedListener = searchFragmentDelegate.onItemViewSelectedListener
+		val rowsSupportFragment = RowsSupportFragment().apply {
+			adapter = searchFragmentDelegate.rowsAdapter
+			onItemViewClickedListener = searchFragmentDelegate.onItemViewClickedListener
+			onItemViewSelectedListener = searchFragmentDelegate.onItemViewSelectedListener
 		}
+
+		childFragmentManager.commit {
+			replace(binding.resultsFrame.id, rowsSupportFragment)
+		}
+
+		return binding.root
+	}
+
+	override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+		super.onViewCreated(view, savedInstanceState)
 
 		viewModel.searchResultsFlow
 			.onEach { searchFragmentDelegate.showResults(it) }
 			.launchIn(lifecycleScope)
+
+		val query = arguments?.getString(SearchFragment.EXTRA_QUERY)
+		if (!query.isNullOrBlank()) {
+			binding.searchBar.setText(query)
+			viewModel.searchImmediately(query)
+			binding.resultsFrame.requestFocus()
+		} else {
+			binding.searchBar.requestFocus()
+		}
 	}
 
 	override fun onDestroyView() {

--- a/app/src/main/java/org/jellyfin/androidtv/ui/startup/StartupActivity.kt
+++ b/app/src/main/java/org/jellyfin/androidtv/ui/startup/StartupActivity.kt
@@ -1,6 +1,7 @@
 package org.jellyfin.androidtv.ui.startup
 
 import android.Manifest
+import android.app.SearchManager
 import android.content.Intent
 import android.os.Bundle
 import android.widget.Toast
@@ -137,7 +138,9 @@ class StartupActivity : FragmentActivity() {
 		// Create destination
 		val destination = when {
 			// Search is requested
-			intent.action === Intent.ACTION_SEARCH -> Destinations.search
+			intent.action === Intent.ACTION_SEARCH -> Destinations.search(
+				query = intent.getStringExtra(SearchManager.QUERY)
+			)
 			// User view item is requested
 			itemId != null && itemIsUserView -> {
 				val item by api.userLibraryApi.getItem(itemId = itemId)

--- a/app/src/main/res/layout/fragment_search_text.xml
+++ b/app/src/main/res/layout/fragment_search_text.xml
@@ -48,9 +48,8 @@
         app:layout_constraintTop_toTopOf="@id/logo" />
 
     <!-- Fragment showing search results -->
-    <androidx.fragment.app.FragmentContainerView
+    <FrameLayout
         android:id="@+id/results_frame"
-        android:name="androidx.leanback.app.RowsSupportFragment"
         android:layout_width="0dp"
         android:layout_height="0dp"
         android:layout_marginTop="27dp"

